### PR TITLE
Finished implementation of Encore GK in CM UI

### PR DESCRIPTION
**Pull request by kyanokashi**
Finished implementing the UI. Have the UI listening to the gateway token address and launch the mint after a successful verification.

### CODE REVIEW

**Review by Buzzec — APPROVED** [review]
Good from the identity side

**Comment by stegaBOB** [comment]
Please fix the conflicts. Thanks

**Comment by kyanokashi** [comment]
Sycned with the latest metaplex repo

**Comment by kyanokashi** [comment]
I merged the latest metaplex repo

File: .gitignore [hunk 1]
```diff
@@ -34,7 +34,7 @@ hfuzz_workspace
 **/.DS_Store
 .cache
 js/packages/web/.env
-js/packages/candy-machine-ui/.env
+js/packages/candy-machine-ui/.env.example
```
Comment (stegaBOB) [inline .gitignore:37]: it needs to be the reverse. The .example should be added but the normal .env should be ignored

Comment (kyanokashi) [reply .gitignore:37]: yea my bad didn't notice

**Review by stegaBOB — APPROVED** [review]

**Pull request merged**